src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -260,124 +261,132 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
     }>
     private _middlewares: Middleware[] = []
     private _engineMiddlewares: EngineMiddleware[] = []
+    private _clientVersion: string
     constructor(optionsArg?: PrismaClientOptions) {
-      const options: PrismaClientOptions = optionsArg ?? {}
-      const internal = options.__internal ?? {}
-
-      const useDebug = internal.debug === true
-      if (useDebug) {
-        Debug.enable('prisma-client')
-      }
-
-      if (internal.hooks) {
-        this._hooks = internal.hooks
-      }
+      this._clientVersion = config.clientVersion ?? clientVersion
```

Comment:
The client version is passed from the generated client in the config object. I'm not sure why the field is optional though so I'm fallbacking to the local package.json version in case it's not there.

Reply:
We could probably make it required, this is rather an old artifact, as in some tests we sometimes didn't have it.